--- a/setup.py	2018-10-19 08:52:24.000000000 -0500
+++ b/setup.py	2019-09-12 11:44:02.321054280 -0500
@@ -79,11 +79,11 @@
 
 data_files = [
     (
-        'etc/jupyter/jupyter_notebook_config.d',
+        '/etc/jupyter/jupyter_notebook_config.d',
         [pjoin('etc', 'ipyparallel-serverextension.json')],
     ),
     (
-        'etc/jupyter/nbconfig/tree.d',
+        '/etc/jupyter/nbconfig/tree.d',
         [pjoin('etc', 'ipyparallel-nbextension.json')],
     ),
     (
